internal/pkg/template/workload.go

```diff
@@ -510,6 +510,7 @@ type WorkloadOpts struct {
 	HTTPVersion              *string
 	ALBEnabled               bool
 	HostedZoneAliases        AliasesForHostedZone
+	CredentialsParameter     string
```

Comment:
qq: perhaps i missed something - what is this for?

Reply:
The parameter already existed at l 527, I just moved it up :D 

It's the private registry creds I believe

Reply:
woop me blind!